src/Api.js

```diff
@@ -89,13 +91,100 @@ class Api {
 	}
 
 	/**
-	 * Get the WhoColor data for a given wiki page.
+	 * Extract token and editor IDs from a WikiWho span element with `id='token-X'` and
+	 * `class='token-editor-Y'` attributes.
 	 *
-	 * @param  {string} wikiUrl URL of the wiki page that we want to analyze.
+	 * @param {HTMLElement} element
+	 * @return {*[]} A two-element array: 0 = int token_id, 1 = string editor_id.
+	 */
+	getIdsFromElement( element ) {
+		const out = [ false, false ],
+			tokenMatches = element.id.match( /token-(\d+)/ ),
+			editorMatches = element.className.match( /token-editor-([^\s]+)/ );
+		if ( tokenMatches && tokenMatches[ 1 ] ) {
+			out[ 0 ] = parseInt( tokenMatches[ 1 ] );
+		}
+		if ( editorMatches && editorMatches[ 1 ] ) {
+			out[ 1 ] = editorMatches[ 1 ];
+		}
+		return out;
```

Comment:
Is there a reason why we want to keep using arrays rather than a readable object with keys?

It's not really crucial, but since this is what **we** make (rather than what we get from the WikiWho api) we can start making things more readable for maintainability and readability.

I don't think this is a merge-preventer, but I noticed it and thought we should consider these things. The fact that WikiWho gives us unintelligible array-stuff made me a bit more sensistive to arrayed results :p